setup.py

```diff
@@ -17,55 +17,80 @@
 import sys
 from distutils.core import setup
 from setuptools import find_packages
+from setuptools.command.install import install
+
 
 DESCRIPTION = "Icetea - test framework"
 OWNER_NAMES = 'Jussi Vatjus-Anttila'
 OWNER_EMAILS = 'jussi.vatjus-anttila@arm.com'
+VERSION = "1.0.1"
+
+
+def read(fname):
+    """
+    Utility function to cat in a file
+    :param fname: filename
+    :return: file content as a String
+    """
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
-INSTALL_REQUIRES = [
-          "prettytable",
-          "requests",
-          "yattag",
-          "pyserial>2.5",
-          "jsonmerge",
-          "mbed-ls>=1.5.1,==1.*",
-          "semver",
-          "mbed-flasher==0.9.*",
-          "six"
-      ]
+
+INSTALL_REQUIRES = read("requirements.txt").splitlines()
+TEST_REQUIRES = read("dev_requirements.txt").splitlines()
+# todo move pyshark as optional dep:
```

Comment:
Should pyshark be mentioned in the documentation, as it is not installed by running installation of dependencies from the requirements.txt?

Reply:
I don't change pyshark installation yet away of default requirements - that require additional work in implementation side. -> follow-up PR